cmake/thirdparty/SetupSeracThirdParty.cmake

```diff
@@ -203,9 +203,12 @@ if (NOT SERAC_THIRD_PARTY_LIBRARIES_FOUND)
 
         # Patch the mfem target with the correct include directories
         get_target_property(_mfem_includes mfem INCLUDE_DIRECTORIES)
-        target_include_directories(mfem SYSTEM INTERFACE $<BUILD_INTERFACE:${_mfem_includes}>)
+        target_include_directories(mfem SYSTEM INTERFACE ${_mfem_includes})
```

Comment:
@chapman39 This was the cause of the weird include directories we were seeing friday.  I cannot explain how something that worked for 16 months just now started failing or how this was the cause.

For people that missed the fun, this was taking a list of include directories and making them system includes and inherited down the chain but magically started tacking on the `PROJECT_SOURCE_DIR` in a way that broke it.  For example:

```
/usr/workspace/white238/serac/repo<$BUILD_INTERFACE:/some;/list;/of;/includes
```

when it should have just been this:

```
<$BUILD_INTERFACE:/some;/list;/of;/includes
```